Formula/samba.rb

```diff
@@ -40,12 +39,21 @@ class Samba < Formula
     depends_on "openssl@1.1"
   end
 
+  resource "libntirpc" do
+    url "https://github.com/nfs-ganesha/ntirpc/archive/refs/tags/v4.0.tar.gz"
+    sha256 "6a7921cd48d02077b394a02be42bf4f9b4678d4ebfd86a7ae4385bbcdce55462"
+  end
+
   resource "Parse::Yapp" do
     url "https://cpan.metacpan.org/authors/id/W/WB/WBRASWELL/Parse-Yapp-1.21.tar.gz"
     sha256 "3810e998308fba2e0f4f26043035032b027ce51ce5c8a52a8b8e340ca65f13e5"
   end
 
   def install
+    resource("libntirpc").stage do
+      include.install Dir["ntirpc/rpc/*"]
+    end
```

Comment:
Let's not install these headers into Samba's prefix.

CC @Homebrew/linux, since we'll probably need `libntirpc` for other formulae too.

Reply:
I have a branch that compiles ntirpc, I'll open a pull request for it momentarily.